<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADPCMSampleEditor</class>
 <widget class="QWidget" name="ADPCMSampleEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QSpinBox" name="sampleNumSpinBox">
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string notr="true">#</string>
       </property>
       <property name="maximum">
        <number>127</number>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="usersLabel">
       <property name="text">
        <string>Users:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usersLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="memoryLabel">
       <property name="text">
        <string>Memory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="memoryWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="sampleViewWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="horizontalScrollBar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QCheckBox" name="repeatCheckBox">
       <property name="text">
        <string>Repeat</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="detailLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string notr="true">, x,</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="rootKeyGroupBox">
     <property name="title">
      <string>Root key</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="rootKeyLabel">
        <property name="text">
         <string>Key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="rootKeyComboBox">
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>E</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A#</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="rootKeySpinBox">
        <property name="maximum">
         <number>7</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rootRateLabel">
        <property name="text">
         <string>Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="rootRateSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="action_Resize">
   <property name="icon">
    <iconset resource="../../bamboo_tracker.qrc">
     <normaloff>:/icon/settings</normaloff>:/icon/settings</iconset>
   </property>
   <property name="text">
    <string>&amp;Resize</string>
   </property>
  </action>
  <action name="actionRe_verse">
   <property name="icon">
    <iconset resource="../../bamboo_tracker.qrc">
     <normaloff>:/icon/reverse</normaloff>:/icon/reverse</iconset>
   </property>
   <property name="text">
    <string>Re&amp;verse</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="../../bamboo_tracker.qrc">
     <normaloff>:/icon/zoom_in</normaloff>:/icon/zoom_in</iconset>
   </property>
   <property name="text">
    <string>Zoom I&amp;n</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="../../bamboo_tracker.qrc">
     <normaloff>:/icon/zoom_out</normaloff>:/icon/zoom_out</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
  </action>
  <action name="action_Import">
   <property name="icon">
    <iconset resource="../../bamboo_tracker.qrc">
     <normaloff>:/icon/load_inst</normaloff>:/icon/load_inst</iconset>
   </property>
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="icon">
    <iconset resource="../../bamboo_tracker.qrc">
     <normaloff>:/icon/duplicate_order</normaloff>:/icon/duplicate_order</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="action_Grid_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../bamboo_tracker.qrc">
     <normaloff>:/icon/eye</normaloff>:/icon/eye</iconset>
   </property>
   <property name="text">
    <string>&amp;Grid View</string>
   </property>
  </action>
  <action name="actionG_rid_Settings">
   <property name="text">
    <string>Grid &amp;Settings...</string>
   </property>
  </action>
  <action name="action_Draw_Sample">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../bamboo_tracker.qrc">
     <normaloff>:/icon/edit_inst</normaloff>:/icon/edit_inst</iconset>
   </property>
   <property name="text">
    <string>&amp;Draw Sample</string>
   </property>
  </action>
  <action name="actionDirec_t_Draw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../bamboo_tracker.qrc">
     <normaloff>:/icon/chart_edit</normaloff>:/icon/chart_edit</iconset>
   </property>
   <property name="text">
    <string>Direc&amp;t Draw</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>sampleNumSpinBox</tabstop>
  <tabstop>usersLineEdit</tabstop>
  <tabstop>repeatCheckBox</tabstop>
  <tabstop>rootKeyComboBox</tabstop>
  <tabstop>rootKeySpinBox</tabstop>
  <tabstop>rootRateSpinBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../bamboo_tracker.qrc"/>
 </resources>
 <connections/>
</ui>
